package.json

```diff
@@ -46,7 +46,7 @@
     "csv": "^1.1.0",
     "debug": "^4.1.0",
     "optimist": "^0.6.1",
-    "sphere-node-sdk": "^2.1.2",
+    "sphere-node-sdk": "^2.3.0",
```

Comment:
`sphere-stock-import` uses `sphere-node-sdk` for the inventory import.
`sphere-node-sdk` is updated because new feature has been added that create and update `restockableInDays` value.